autogtp/sprt.cpp

```diff
@@ -0,0 +1,210 @@
+/*
+    This file is part of Leela Zero.
+    Copyright (C) 2017 Marco Calignano
+    originally taken from Cute Chess (http://github.com/cutechess)
+    Copyright (C) 2016 Ilari Pihlajisto
+
+    Leela Zero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Leela Zero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Leela Zero.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "sprt.h"
+#include <cmath>
+#include <QtGlobal>
+
+class BayesElo;
+class SprtProbability;
+
+class BayesElo
+{
+	public:
+		BayesElo(double bayesElo, double drawElo);
```

Comment:
Gives that Go rarely has draws this can be dramatically simplified.

I've uploaded an old python implementation I used here
https://gist.github.com/sethtroisi/786b99e37f64cd034ff0baec649ca2b5

If you don't want to do this and are willing I'll jump in tomorrow and write a c++ version

Reply:
He doesn't need a C++ version, he already has one.